android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -470,6 +470,18 @@ fun PlayerEvent.toJson(): WritableMap {
             json.putString("deviceName", deviceName)
         }
 
+        is PlayerEvent.CueEnter -> {
+            json.putDouble("start", start)
+            json.putDouble("end", end)
+            json.putString("text", text)
+        }
+
+        is PlayerEvent.CueExit -> {
+            json.putDouble("start", start)
+            json.putDouble("end", end)
+            json.putString("text", text)
+        }
```

Comment:
Does it make sense to relay all cue events even if they don't make sense with the supported content? E.g. bitmap subtitles will not have any useful content for customers without the image IMO.

Reply:
That's a valid point, thank you for bringing it up. 👍    
But the idea for this PR was to implement the same baseline API we did for the Flutter SDK. So supporting other `Cue` fields would be out of scope here.  
Filtering the cue events which are not supported yet could be object of discussion, but I think the plan for now is to keep it simple.  

Let me know however if you think we should go that direction with this PR 🙌

Reply:
> supporting other Cue fields would be out of scope here.
Makes sense. 👍 

> the plan for now is to keep it simple.

Fine for me. Should we add some documentation around what is supported and what now? I still think it is confusing to get `Cue`s without any useful information.

Reply:
Nice idea. I added a note in ca321df922423048775deb723bc9b567e9f6ba7a